src/constants/chainInfo.ts

```diff
@@ -241,6 +243,38 @@ const CHAIN_INFO: ChainInfoMap = {
     color: darkTheme.chain_43114,
     backgroundColor: darkTheme.chain_43114_background,
   },
+  [ChainId.BASE]: {
+    networkType: NetworkType.L2,
+    blockWaitMsBeforeWarning: ms`25m`,
+    bridge: 'https://bridge.base.org/deposit',
```

Comment:
this is probably more for the Base team but this is auto-routing to goerli currently, when they go live this will be the right url?